### PR TITLE
Removed TOML ruptures

### PR DESCRIPTION
**Pull request by micheles**
Because now, after https://github.com/gem/oq-engine/issues/5603, we can use CSV instead of TOML.

### CODE REVIEW

File: doc/adv-manual/special-features.rst [hunk 1]
```diff
@@ -142,28 +142,23 @@ PointSources/AreaSources/MultiPointSources, fault sources are
...
+Ruptures in CSV format
 -------------------------------------------
 
-Since engine v3.8 there is a way to serialize ruptures in
-TOML_ format. The command to give is::
+Since engine v3.10 there is a way to serialize ruptures in
+CSV format. The command to give is::
   
-  $ oq extract rupture/<rup_id> <calc_id>`
+  $ oq extract ruptures?min_mag=<mag> <calc_id>`
```
Comment (raoanirudh) [inline doc/adv-manual/special-features.rst:151]: On zsh (default shell on macOS), the query string needs to be quoted:
`$ oq extract "ruptures?min_mag=<mag>" <calc_id>`

File: doc/adv-manual/special-features.rst [hunk 2]
```diff
@@ -142,28 +142,23 @@ PointSources/AreaSources/MultiPointSources, fault sources are
...
-extension is `.txt` so that it can be open with any text editor, but the
-format is actually TOML (the engine will accept both ``.txt`` and ``.toml``
-extensions). Then it is possible to run scenario
+we can extract the ruptures in the datastore with magnitude larger than
+6 with ``oq extract ruptures?min_mag=6 42``: this will generate a CSV file.
+Then it is possible to run scenario
 calculations starting from that rupture by simply setting
 
-``rupture_model_file = rupture-0_42.txt``
+``rupture_model_file = ruptures-min_mag=6_42.csv``
```
Comment (raoanirudh) [inline doc/adv-manual/special-features.rst:159]: Need to confirm if the `=` character is permitted for Windows filenames.

Comment (micheles) [reply doc/adv-manual/special-features.rst:159]: `=` is valid, the reserved characters are

        < (less than)
        > (greater than)
        : (colon)
        " (double quote)
        / (forward slash)
        \ (backslash)
        | (vertical bar or pipe)
        ? (question mark)
        * (asterisk)

File: doc/adv-manual/special-features.rst (hunk 1 above)
Comment (micheles) [reply doc/adv-manual/special-features.rst:151]: Fixed the docs

**Pull request merged**